core/services/relay/evm/mercury/queue.go

```diff
@@ -72,6 +72,24 @@ func NewTransmitQueue(lggr logger.Logger, feedID string, maxlen int) *TransmitQu
 	}
 }
 
+// InitTransmissions initializes the priority queue with the given transmissions. Any
+// previously pushed transmissions are dropped.
+func (tq *TransmitQueue) InitTransmissions(transmissions []*Transmission) {
```

Comment:
Why not pass it as an argument into `NewTransmitQueue` and cast to `pq` directly?

Reply:
Originally I didn't want `mt.queue` to be nil before Start is called and we instantiate a new queue. But it looks like it's only used in `HealthReport` outside of start/close/runloop. Is it possible that HealthReport is called before Start?